TTS/utils/io.py

```diff
@@ -48,12 +49,23 @@ def copy_model_files(config, out_path, new_fields):
             )
 
 
+def load(path, **kwargs):
```

Comment:
Maybe renaming them as `load_fsspec` and `save_fsspec` makes more sense ?

Reply:
sure, that makes sense 🙂